### DISCUSSION TITLE
Cypress version >= 7.2.0 takes 5-10 mins to load

### DISCUSSION THREAD

**Discussion by nthali** [cypress-io/cypress#17041] [General]
We have found that Cypress version 7.2.0 and later loads extremely slowly. It takes at least 5-10 mins for the first test (we are running feature files) to run, so we are deliberately still using 7.0.0 or 7.1.0. Anyone else dealing with this issue?
Thanks,
Nilesh

**Comment by jennifer-shehane** [comment]
@nthali We're not aware of any regression in terms of loading time of Cypress in 7.2.0. If you can provide a repo we could run to recreate this, it would be greatly appreciated. 🙏

**Reply by nthali** [reply]
@jennifer-shehane thanks for responding. i was afraid you were going to ask for a repo :) since the code is client protected, i can't provide a repo. let me ask this: i'm assuming you guys have some sort of load time regression test for your test apps and you haven't seen any noticeable difference?
Thanks!
